app/assets/stylesheets/styles/components/guides/_guides.css.scss

```diff
@@ -95,87 +161,192 @@
-    padding: .2rem 1rem;
+    padding: 0.2rem 1rem;
 
     @include accordion-header-color($of-orange);
 
     h2, h5 {
+      color: #333;
       display: inline-block;
-      font-weight: 100;
+      font-weight: 600;
+      margin: 0.3rem 0;
     }
 
     .title {
-      font-size: 1.5rem;
+      font-size: 1.8rem;
     }
 
     .author {
-      margin-left: 20px;
       display: inline-block;
+      margin-left: 20px;
     }
   }
 
   .accordion dd > .content {
-    padding: 0 0.935rem;
+    padding: 0;
+  }
+
+  .accordion dd.accordion-navigation {
+    margin-top: 2rem;
   }
 
   .actions {
-    border: 1px solid #CCCCCC;
+    margin-top: 1rem;
 
     .action {
-      padding: 10px 40px;
-      border-top: 2px solid #333333;
+      border-top: 1px solid rgb(148, 148, 148);
+      padding: 1.5rem;
     }
   }
 
   /* Stages in completed guides */
-  .stages{
-    .image img{
-      height: 170px;
+  .stages {
+
+    // This is shared with guide creation steps, and should be DRYed up
+
+    .stage-wrapper {
+      display: inline-block;
+      height: 300px;
+      margin-bottom: 3%;
+      overflow: hidden;
+    }
+
+    .stage-option {
+      border: solid 1px #dfdfdf;
+      border-radius: 4px;
+      display: block;
+      height: 100%;
+      padding: 0.25rem;
+      width: 100%;
+    }
+
+    .stage-option, .chosen-stage > .bubble {
+      background-image: url('stages/other_sm.png');
+      background-position: center center;
+      background-repeat: no-repeat;
       background-size: cover;
+
+
+      &.Preparation {
+        background-image: url('stages/prep_sm.png');
+      }
+
+      &.Sow {
+        background-image: url('stages/sow_sm.png');
+      }
+
+      &.Germination {
+        background-image: url('stages/germination_sm.png');
+      }
+
+      &.Seedling {
+        background-image: url('stages/sprouting_sm.png');
+      }
+
+      &.Juvenile {
+        background-image: url('stages/juvenile_sm.png');
+      }
+
+      &.Adult {
+        background-image: url('stages/adult_sm.png');
+      }
+
+      &.Flowering {
+        background-image: url('stages/flowering_sm.png');
+      }
+
+      &.Fruit {
+        background-image: url('stages/fruiting_sm.png');
+      }
+
+      &.Dormant {
+        background-image: url('stages/dormant_sm.png');
+      }
+    }
+
+    // End need DRYing
+
+    .action-content {
+      border-bottom: #cdcdcd 2px solid;
+      margin-top: 1rem;
+    }
+
+    .action-content:last-child {
+      border-bottom: none;
+    }
+
+    .image img {
       background-position: center;
+      background-size: cover;
+      height: 170px;
       width: 100%;
     }
 
-    h3{
+    h3 {
       font-size: 1rem;
       margin: 0;
     }
 
-    /* Action labels in completed guides look different */
+    .tabs {
+      background-color: #efefef;
+    }
+
+    .tabs-content {
+      background-color: #f6f6f6;
+    }
+
+    .tabs dd > a {
+      padding: 0.75rem 1.5rem;
+    }
+
+    .tabs > dd.active > a,
```

Comment:
Avoid qualifying class selectors with an element.<br>Selector should have depth of applicability no greater than 2, but was 7